### DISCUSSION TITLE
Can someone help me to  modify this lydrc file?

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#858] [General]
[asap7.lydrc.txt](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/files/10938510/asap7.lydrc.txt)

Error message:
![image](https://user-images.githubusercontent.com/127094304/224223678-4b9d9b18-a448-4d50-adb4-0fb10730554f.png)
![image](https://user-images.githubusercontent.com/127094304/224223708-247208dc-5c89-4948-901e-4a349458a17f.png)
![image](https://user-images.githubusercontent.com/127094304/224223724-f7b9596a-dd8c-45c8-89c3-3b8135967d6e.png)

**Comment by yathAg** [comment]
I was able to get DRC for asap7 working, and [here](https://github.com/yathAg/openroad_vsd_7nmContest/tree/test/Files/asap7/drc) is the file I used 

I have also documented some steps [here](https://github.com/yathAg/openroad_vsd_7nmContest/tree/test#understanding-drc-using-klayout)

**Reply by yathAg** [reply]
Also, please check your KLayout version and make sure its greater than 0.27.1

**Reply by its-rakib** [reply]
It would be great help for me if you can provide me the .lylvs file also .@yathAg

**Reply by yathAg** [reply]
Unfortunately, the drc files were taken from [https://github.com/laurentc2/ASAP7_for_KLayout](https://github.com/laurentc2/ASAP7_for_KLayout), and the lvs files haven't been written yet.
